website/docs/d/cos_replication.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Object Storage"
+layout: "ibm"
+page_title: "IBM : Cloud Object Storage Replication"
```

Comment:
Is this Cloud Object Storage Replication or Cloud Object Storage bucket Replication